state-chain/chains/src/sol.rs

```diff
@@ -210,3 +196,11 @@ impl address::ToHumanreadableAddress for SolAddress {
 		self.to_string()
 	}
 }
+
+#[derive(Clone, Encode, Decode, TypeInfo, Eq, PartialEq, RuntimeDebug)]
+pub struct SolanaDepositChannelState {
+	pub seed: Vec<u8>,
```

Comment:
(looks like it's just the channel_id - in that case can we not just use u64?).

Reply:
the seeds in Solana are of variable length up to 32 bytes. We could use some fixed length if we wanted as long as we make sure we encode it correctly in the api call.

Reply:
I realized that since the seed is just channel ID, we can only use the "bump" u8 as the ChannelState. We can get the seed from the channel ID when we need to.